.github/workflows/regression_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Origen App Generator Regression Tests
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        ruby-version: [2.5, 2.6]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+    - name: Update Env
+      run: echo "ACTIONS_ALLOW_UNSECURE_COMMANDS=true" >> $GITHUB_ENV
+    - name: Install Bundler
+      run: gem install bundler -v '< 2'
+    - name: Install dependencies
+      run: bundle install  
+    - name: Gem Install Origen 
+      run: gem install origen 
+    - name: Setup Dependencies
+      run: origen -v
+    - name: Set Origen to debug mode
+      run: origen m debug
+    - name: Run Spec tests
+      run: origen app_gen:test --regression
+    - name: Verify Building website
+      run: origen web compile --no-serve
```

Comment:
dont think app generators has web compile

Reply:
Yep fixed in latest commit